Mu2eG4/geom/bfgeom_DS70_no_tsu_ps_v01.txt

```diff
@@ -23,6 +23,7 @@ vector<string> bfield.innerMaps = {
   "BFieldMaps/Mau13/DSExtension.header"
 };
 
+double bfield.scaleFactor = 0.7;
```

Comment:
Can you explain what you are doing here.  I think that this will make a field in the DS that is 0.49 times the nominal field?  The factor multiples the existing map.

Reply:
I looked at the code you pointed to in your comment.   I like your suggestion of looking up a nominal value in the map rather than using this scale factor. 

I am 99% sure that the PR as is should be rejected.  I will wait to hear from Dave and I added him as a reviewer.